ereuse_devicehub/resources/action/views.py

```diff
@@ -1,9 +1,13 @@
+import os
+import json
+from time import time
 from distutils.version import StrictVersion
 from typing import List
 from uuid import UUID
 
 from flask import current_app as app, request, g
 from sqlalchemy.util import OrderedSet
+from marshmallow.exceptions import ValidationError as mValidationError
```

Comment:
Is that an unused import?

Reply:
* imports unused removed
* TMP_SNAPSHOT defined in config or in env file as you prefere
* email added to the name of file of json
* date more readable in the name of file of json